runbotics-orchestrator/src/main/java/com/runbotics/service/criteria/ProcessCollectionCriteria.java

```diff
@@ -24,6 +24,8 @@ public class ProcessCollectionCriteria implements Serializable, Criteria {
 
     private UUIDFilter parentId;
 
+    public ProcessCollectionCriteria() {}
```

Comment:
empty?

Reply:
It is necessary, other way it won't work. It might be something to preprocess by some dependency in project.